src/utils/vc/presentations.js

```diff
@@ -123,8 +123,9 @@ export async function verifyPresentation(presentation, options = {}) {
   // Build verification options
   const verificationOptions = {
     suite: [new Ed25519Signature2018(), new EcdsaSepc256k1Signature2019(), new Sr25519Signature2020()],
-    documentLoader: defaultDocumentLoader(resolver),
+    documentLoader: options.documentLoader || defaultDocumentLoader(resolver),
```

Comment:
One last thing. You think I should add that assertion here since it resolver is potentially being ignored here too?

Reply:
yeah good spot